init.c

```diff
@@ -253,52 +253,55 @@ static void print_help()
 	  "Usage: mktorrent [OPTIONS] <target directory or filename>\n\n"
 	  "Options:\n"
 #ifdef USE_LONG_OPTIONS
-	  "-a, --announce=<url>[,<url>]* : specify the full announce URLs\n"
-	  "                                additional -a adds backup trackers\n"
-	  "-c, --comment=<comment>       : add a comment to the metainfo\n"
-	  "-d, --no-date                 : don't write the creation date\n"
-	  "-f, --force                   : overwrite output file if it exists\n"
-	  "-h, --help                    : show this help screen\n"
-	  "-l, --piece-length=<n>        : set the piece length to 2^n bytes,\n"
-	  "                                default is calculated from the total size\n"
-	  "-n, --name=<name>             : set the name of the torrent\n"
-	  "                                default is the basename of the target\n"
-	  "-o, --output=<filename>       : set the path and filename of the created file\n"
-	  "                                default is <name>.torrent\n"
-	  "-p, --private                 : set the private flag\n"
-	  "-s, --source=<source>         : add source string embedded in infohash\n"
+	  "-a, --announce=<url>[,<url>]*    : specify the full announce URLs\n"
+	  "                                   additional -a adds backup trackers\n"
+	  "-c, --comment=<comment>          : add a comment to the metainfo\n"
+	  "-d, --no-date                    : don't write the creation date\n"
+	  "-e, --exclude=<match>[,<match>]* : exclude files based on filename pattern\n"
```

Comment:
I think `<pattern>` would be better, or even just `<pat>` / `<p>` so that it's not very long, and then you can say something like "exclude files whose name matches the pattern `<p>` ...", etc.